skills/enso/__init__.py

```diff
@@ -12,6 +13,8 @@ def get_enso_skill(
 
     if name == "get_tokens":
         return EnsoGetTokens(api_token=api_token, main_tokens=main_tokens, store=store, agent_id=agent_id)
+    if name == "get_route":
```

Comment:
Name consistency, In addition to config name, class name, tool name, there's also file name.